schemas/io-package.json

```diff
@@ -65,13 +80,22 @@
                     "description": "true if messagebox supported. If yes, the object system.adapter.<instance>.messagebox will be created to send messages to adapter",
                     "type": "boolean"
                 },
+                "readme": {
+                    "description": "Path to the ReadMe file",
```

Comment:
relative local file path or absolute URL?

Reply:
url works for sure, path not sure, will better change it